package.json

```diff
@@ -41,6 +41,7 @@
 		"start": "npm ci && newspack-scripts watch",
 		"watch": "newspack-scripts watch",
 		"test": "newspack-scripts test",
+		"tsc": "tsc --watch",
```

Comment:
Also not a blocker, just curious: how does this differ from the regular `watch` script? It looked to me like the TS is already getting compiled in that existing script.

Reply:
I found, that while the existing watch task does compile TS, it doesn't catch any issues until they hit CI. Having this essentially mimics the behavior of the [`newspack/check-typescript` job](https://github.com/Automattic/newspack-plugin/blob/trunk/.circleci/config.yml#L17) and displays any issues into the terminal.